releases/2023.1/2023.1.md

```diff
@@ -0,0 +1,172 @@
+
+```
+pip-tools >= 2.0
+pytest==7.2.1
+pytest-cov==4.0.0
+pytest-asyncio==0.20.3
+black==23.3.0
+isort==5.12.0
+pylint==2.15.0
+pycodestyle==2.10.0
+yala==3.2.0
+tox==3.28.0
+virtualenv==20.21.0
+typing-extensions==4.5.0
+```
+## Kytos-ng and NApps released
+
+The following Kytos-ng projects and NApps were released in this version. For further detailed information about what was released and what changed, please visit the release page of each project you are interested in knowing more about and look for the changelog file.
+
+
+Project                                                        | Release Web Page
+-------------------------------------------------------------- |------------------------------------------------------------------------
+[kytos](https://github.com/kytos-ng/kytos)                     | [kytos](https://github.com/kytos-ng/kytos/releases)
+[python-openflow](https://github.com/kytos-ng/python-openflow) | [python-openflow](https://github.com/kytos-ng/python-openflow/releases)
+[kytos-utils](https://github.com/kytos-ng/kytos-utils)         | [kytos-utils](https://github.com/kytos-ng/kytos-utils/releases)
+[ui](https://github.com/kytos-ng/ui)                           | [ui](https://github.com/kytos-ng/ui/releases)
+[kytos/of_core](https://github.com/kytos-ng/of_core)           | [kytos/of_core](https://github.com/kytos-ng/of_core/releases)
+[kytos/flow_manager](https://github.com/kytos-ng/flow_manager) | [kytos/flow_manager](https://github.com/kytos-ng/flow_manager/releases)
+[kytos/topology](https://github.com/kytos-ng/topology)         | [kytos/topology](https://github.com/kytos-ng/topology/releases)
+[kytos/of_lldp](https://github.com/kytos-ng/of_lldp)           | [kytos/of_lldp](https://github.com/kytos-ng/of_lldp/releases)
+[kytos/pathfinder](https://github.com/kytos-ng/pathfinder)     | [kytos/pathfinder](https://github.com/kytos-ng/pathfinder/releases)
+[amlight/coloring](https://github.com/kytos-ng/coloring)       | [kytos-ng/coloring](https://github.com/kytos-ng/coloring/releases)
+[amlight/sdntrace](https://github.com/kytos-ng/sdntrace)       | [kytos-ng/sdntrace](https://github.com/kytos-ng/sdntrace/releases)
+[amlight/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp) | [kytos-ng/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp/releases)
+[amlight/kytos_stats](https://github.com/kytos-ng/kytos_stats) | [kytos-ng/flow_stats](https://github.com/kytos-ng/kytos_stats/releases)
+[amlight/flow_stats](https://github.com/kytos-ng/flow_stats)   | [kytos-ng/flow_stats](https://github.com/kytos-ng/flow_stats/releases)
+[kytos/mef_eline](https://github.com/kytos-ng/mef_eline)       | [kytos/mef_eline](https://github.com/kytos-ng/mef_eline/releases)
+
+<!---
+TODO remove this comment and move maintenance to the table above once 2023.1.1 is released
```

Comment:
cc'ing @Ktmi for his info, so the idea that I was hoping was to include this back again on the table above once `epic_maintenance_v1` is delivered and then we can also augment the rest of the release notes in a new issue on #56, including a general objective for `maintenance` and briefly summarizing what changed, wdyt?